### PR TITLE
Appendix_meta_rules fast review

### PR DESCRIPTION
**Pull request by frack113**
- Add "generate" field
- Change to filter
- Small fix

Need review.

### CODE REVIEW

File: appendix_meta_rules.md [hunk 1]
```diff
@@ -93,8 +93,10 @@ This was the first approach defined in Sigma with aggregations and the near oper
 
 ## Type of rules
 ### Correlation rules
+
 The purpose is to cover a detection like:
-* X invalid login alerts on a host
+
+* X invalid login alerts on an unique host
```
Comment (phantinuss) [inline appendix_meta_rules.md:99]:
```suggestion
* X invalid login alerts on a unique host
```

File: appendix_meta_rules.md [hunk 2]
```diff
@@ -105,8 +107,9 @@ For this reason, another document type is included for file inclusion.
 An inclusion can be defined by setting the action attribute to include.  
 All rules contained in the referenced file are handled as if they were defined in the including file.
 
-###  Defeats rules
-The purpose of defeats rules is to cover the same tuning for many rules. They are used to suppress matches of multiple rules. This is most commonly useful for environment specific tuning where a false positive prone application is used in an organization and it's false positives are accepted.
+###  Filter rules
+
+The purpose of Filter rules is to cover the same tuning for many rules. They are used to suppress matches of multiple rules. This is most commonly useful for environment specific tuning where a false positive prone application is used in an organization and it's false positives are accepted.
```
Comment (phantinuss) [inline appendix_meta_rules.md:112]:
```suggestion
The purpose of Filter rules is to apply the same tuning on many rules with the goal to suppress matches of multiple rules. This is most commonly useful for environment specific tuning where a false positive prone application is used in an organization and its false positives are accepted.
```

File: appendix_meta_rules.md [hunk 3]
```diff
@@ -194,15 +208,20 @@ action: correlation
 
 **Attribute:** rules
 
-Refers to one or multiple Sigma or Correlations rules. Allowing the user to chain multiple correlations together. A rule can be referred to by the `id` or (to be defined) `name` attribute of a Sigma rule. Latter is a human-readable name and improves the readability of correlation rules.
+Refers to one or multiple Sigma or Correlations rules.
+Allowing the user to chain multiple correlations together.
+A rule can be referred to by the `id` of a sigma rule.
+
+Latter is a human-readable name and improves the readability of correlation rules.
+In this case, the tool must be able to manage the human-to-id correspondence automatically.
```
Comment (phantinuss) [inline appendix_meta_rules.md:216]: `Latter`? What does it reference? The name field is not introduced before.

```suggestion
A rule can be referred to by the `id` or `name` of a sigma rule.

`name` is a unique human-readable name and improves the readability of correlation rules.
In this case, the tool must be able to manage the human-to-id translation automatically and the referenced rule name has to be defined in the respective rule.
```

At least that's how I understood this the last time we talked about the topic. Can you verify @thomaspatzke ?

File: appendix_meta_rules.md [hunk 4]
```diff
@@ -194,15 +208,20 @@ action: correlation
...
 
 ```yaml
 title: login brute force
 id: 0e95725d-7320-415d-80f7-004da920fc11
 action: correlation
 rules:
     - 5638f7c0-ac70-491d-8465-2a65075e0d86 # ID of the low firewall rule for action: block
-    - firewall_block
+    - firewall_block  # the internal tools have a lookup table to the correct rule id
```
Comment (phantinuss) [inline appendix_meta_rules.md:224]:
```suggestion
    - firewall_block  # The internal tools have a lookup table to the correct rule `id` by `name`
```

File: appendix_meta_rules.md [hunk 5]
```diff
@@ -374,7 +404,8 @@ timespan: 1h
 ordered: true
 ```
 
-The grouping by the ComputerName field is assumed for the many_failed_logins correlation rule but not for the final correlation.
+Nota:
```
Comment (phantinuss) [inline appendix_meta_rules.md:407]:
```suggestion
Note:
```

File: appendix_meta_rules.md [hunk 6]
```diff
@@ -374,7 +404,8 @@ timespan: 1h
...
+Even if the rule many_failed_logins use a grouping by the "ComputerName" field,the correlation rule only use it own `group-by` "User".
```
Comment (phantinuss) [inline appendix_meta_rules.md:408]:
```suggestion
Even if the rule many_failed_logins groups by the "ComputerName" field, the correlation rule only uses its own `group-by` "User".
```

File: appendix_meta_rules.md [hunk 7]
```diff
@@ -490,7 +521,7 @@ action: include
 filename: other_sigma_rule.yml
 ```
 
-# Defeats
+# Filter
```
Comment (phantinuss) [inline appendix_meta_rules.md:524]:
```suggestion
# Suppression Filter
```
Just an idea for discussion. We already use the word filter for negative matches inside rules. To be able to address the filters by name, we could use "inline filter" and "suppression filter" to clarify if one means a specific one. Saying "filter" in general when the location doesn't matter in the discussion will also be fine but I think sometimes you want to be more specific.

If accepted, I will make changes in more locations, of course.

**Review by thomaspatzke — APPROVED** [review]
Looks fine to me!

**Review by phantinuss — APPROVED** [review]

**Pull request merged**